app/policies/project_policy.rb

```diff
@@ -33,14 +33,20 @@ def unarchive?
   alias ownership? share?
 
   class Scope < Scope
+    def permitted_project_ids
+      teams_projects = current_user.teams.map(&:projects).flatten.pluck(:id)
+      user_projects  = current_user.projects.pluck(:id)
```

Comment:
Can we rename these variables to `team_project_ids` and `user_project_ids` since the content of the arrays are ids?